lib/utils/paths.py

```diff
@@ -36,7 +36,7 @@ def get_resource_dir(name):
 
 
 def get_user_dir(name=None):
```

Comment:
Thank you! Looking good...

But I just notice, that we have this nice method here (get_user_dir) and I wonder, why we did not make use of it.
I think it'd be better to use it, so the next time, we need to change anything, it would be just in one place.

Would you mind to update the code, so that we use get_user_dir instead of calling platformdirs directly in the single files?

Reply:
@bkmgit how do we go on with this branch? Do you want to update the code or should we merge and I will update the code to use `get_user_dir` in a separate branch?

Reply:
Can update later this week, but if it is needed sooner than that, please merge and make further changes as needed.

Reply:
It's totally fine. I just wasn't sure, if you wanted to do it.